jabgui/build.gradle.kts

```diff
@@ -24,6 +24,8 @@ dependencies {
     implementation("org.openjfx:javafx-swing")
     implementation("org.openjfx:javafx-web")
 
+    implementation("com.pixelduke:fxthemes:1.5.1")
```

Comment:
According to https://mvnrepository.com/artifact/com.pixelduke/fxthemes/1.6.0, 1.6.0 is the most recent version. Any reason why not using the latest one?